source/index.html

```diff
@@ -1,12 +1,207 @@
 <!DOCTYPE html>
-<html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Мишка</title>
-  </head>
-  <body>
+<html class="page" lang="ru">
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Мишка</title>
+</head>
+
+<body class="page__body">
+  <header class="page-header">
+    <a class="logo" href="#">
+      <img src="img/icons/logo-mobile.svg" width="86" height="35"
+        alt="Главная страница сайта по продаже вязанных вещей Мишка.">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
```

Comment:
```suggestion
      <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню.</span></button>
```

Просто запомни, что скрин-ридер такая штука, которая не человек, ей воздух не нужен, чтобы говорить, поэтому он читает всё подряд без пауз. Чтобы сделать паузу - он должен видеть точку. Поэтому всё, что мы делаем для скрин-ридера на сайте - должно быть с точкой, чтобы он прочитал это как отдельное предложение.